src/main/java/runge/wordledictionary/WordleGame.java

```diff
@@ -0,0 +1,78 @@
+package runge.wordledictionary;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Random;
+
+public class WordleGame
+{
+    private ArrayList<String> wordList;
+    private String randomWord;
+    private final Random rand = new Random();
+    private ArrayList<String> fiveLetterWords;
+
+
+    public WordleGame() throws IOException
+    {
+        WordleDictionary dictionary = new WordleDictionary();
+        wordList = dictionary.getList();
+
+        fiveLetterWords = new ArrayList<>();
+        for (int i = 0; i < wordList.size(); i++)
+        {
+            if (wordList.get(i).length() == 5)
+            {
+                fiveLetterWords.add(wordList.get(i));
+            }
+        }
+        randomWord = fiveLetterWords.get(rand.nextInt(fiveLetterWords.size()));
+    }
+
+    public CharResult[] guess(String guessWord) throws IOException
+    {
+        //gets random word from wordList
+        CharResult[] status = new CharResult[5];
+
+        for (int i = 0; i < randomWord.length(); i++) //iterate 5 times
+        {
+            if (randomWord.contains("" + guessWord.charAt(i)))
```

Comment:
char guessChar = guessWord.charAt(i);
String guessString = String.valueOf(guessChar);

use guessString on line 38 and guessChar on line 40